bin/dune

```diff
@@ -1,13 +1,27 @@
 (executable
  (name cli)
  (public_name ortac)
+ (modules sites cli)
  (package ortac)
  (libraries
+  dune-site
+  dune-site.plugins
+  dune-private-libs.dune-section
```

Comment:
I would say that having to resort to this would be a problem to report, as the dune documentation states about `implicit_transitive_deps false`:
> We recommend users experiment with this mode and report any problems.